site/src/Librecores/ProjectRepoBundle/Resources/views/Project/settings.html.twig

```diff
@@ -16,6 +16,23 @@
     "assets/js/trumbowyg.min.js"
   %}
     <script src="{{ asset_url }}"></script>
+    <script>
+      $(document).ready(function(){
+      $("#delete-project-text").keyup(function(){
+        var projectName = $("#project_displayName").val();
+        if ($(this).val() != projectName) {
+          $('#delete-project-button').attr('disabled', 'disabled');
+        } else {
+          $('#delete-project-button').removeAttr('disabled');
+        }
+      });
+      var deletebutton = document.getElementById("delete-project-button");
```

Comment:
Put them inside the $(), use jQuery where possible.

Reply:
The replacement with jQuery is not the same in this case. $('#delete-project-button') and document.getElementById("delete-project-button") return a different object. It is simply breaking the code if i do this replacement.

Reply:
It might be a technically a different object, but it should function the same way. If not, where exactly do you see the difference?

Reply:
and use `deleteButton` as variable name (camelCase)

Reply:
If you really need a DOM object use $jqObect[0]. However, this is extremely rare. Your case can use `$jqObject.click(...)`